lib/common_components/dialog_box/location_prompt_dialog.dart

```diff
@@ -1,4 +1,5 @@
-import 'package:at_onboarding_flutter/services/size_config.dart';
+// import 'package:at_onboarding_flutter/services/size_config.dart';
+import '../../services/size_config.dart';
```

Comment:
can we maintain consistency in the import statement, we can give an absolute path instead of relative path ?

Reply:
sure